lua/import/regex.lua

```diff
@@ -3,7 +3,7 @@ local regex = {
   go = [[^\t(\".*\")|^import (\".*\")]],
   java = [[^import\s+((static\s+)?[\w.]+\*?);\s*$]],
   javascript = [[^(?:import(?:[\"'\s]*([\w*{}\n, ]+)from\s*)?[\"'\s](.*?)[\"'\s].*)]],
-  lua = [[^local (\w+) = require\(?[\"'](.*?)[\"']\)?]],
+  lua = [[^local (\w+) = require\(?\s*[\"'](.*?)[\"']\s*\)?]],
```

Comment:
The second `\s*` isn't strictly necessary, but it feels more "correct" for balance reasons. It also technically covers cases like `require( 'foo' )`, for what it's worth.